CMakeLists.txt

```diff
@@ -346,6 +347,40 @@ if (LLAMA_CLBLAST)
     endif()
 endif()
 
+if (LLAMA_HIPBLAS)
+    list(APPEND CMAKE_PREFIX_PATH /opt/rocm)
```

Comment:
ROCm path shouldn't be hardcoded to `/opt/rocm`. It's common to use the env var `ROCM_PATH`  (also `ROCM_HOME` is sometime used). `/opt/rocm` should only be a fallback.

Reply:
I took this from AMD's docs, but they have updated it now: [Using CMake](https://rocm.docs.amd.com/en/latest/understand/cmake_packages.html). Probably because it is not going to work in Windows.

Reply:
I hadn't taken a look at AMD's docs. But they at least internally use `ROCM_PATH` on all the projects that I have seen.

As the CMake config would probably need change anyway for windows, and I don't think a lot of people will be impacted by not using their configured ROCm path, I think it's fine to let it that way for now. But whenever change to CMake config to add support for windows, it would be nice to also add support for one of the `ROCM_PATH`/`HIP_PATH`/`ROCM_HOME` on linux.

Reply:
Seems like the latest docs say to always manually use a CMake prefix for configuring. Guess that makes sense because on Windows, people could install it anywhere.

Reply:
On Windows, you'd instead have `HIP_PATH` set IIRC. But someone would need to check the HIP Windows SDK installation to be sure.